### PR TITLE
Automatic removal of disabled datasource (Improvement)

### PR DESCRIPTION
**Pull request by alexbusu**
This PR aims to improve PR #503 by fixing the source matching with External config source.

### CODE REVIEW

File: src/server/utils/cluster-manager/cluster-manager.ts [hunk 1]
```diff
@@ -63,6 +63,10 @@ function getSourceFromExternal(external: External): string {
   return String(external.source);
 }
 
+function externalContainsSource(external: External, source: string): boolean {
+  return Array.isArray(external.source) ? external.source.indexOf(source) > -1 : String(external.source) === source;
```
Comment (adrianmroz) [inline src/server/utils/cluster-manager/cluster-manager.ts:67]: We need to set ts target to version with `Array.includes`. That's task for another day!

Comment (alexbusu) [reply src/server/utils/cluster-manager/cluster-manager.ts:67]: Yes, I was looking for an `Array` method for this, but could not find.

**Comment by alexbusu** [comment]
Will this be part of `1.24.0` release? Or a patch, at least?

**Comment by alexbusu** [comment]
> Thanks for the fix! It could be a part of 1.24 IMHO.

Awesome! Thank you! :rocket:

**Pull request merged**